improver/ensemble_copula_coupling/constants.py

```diff
@@ -71,6 +71,8 @@
     "relative_humidity": Bounds((0, 1.2), "1"),
     "lwe_thickness_of_precipitation_amount": Bounds((0, 0.5), "m"),
     "lwe_thickness_of_precipitation_amount_in_vicinity": Bounds((0, 0.5), "m"),
+    "thickness_of_rainfall_amount": Bounds((0, 0.5), "m"),
+    "lwe_thickness_of_sleetfall_amount": Bounds((0, 0.5), "m"),
```

Comment:
Do you fancy sorting out the grouping of all of these whilst you are making changes? The list is **not**:
- alphabetical
- sorted by diagnostic type
- sorted by matching thresholds

Perhaps thicknesses with thicknesses, alphabetical in their group, rates with rates, alphabetical in their group etc?
- in any semblance of an order.

Reply:
Like this?